docs/_redirects

```diff
@@ -8,3 +8,5 @@
 /reference/rules/javascript_express_insecure_template_rendering /reference/rules/javascript_lang_raw_html_using_user_input
 /reference/rules/javascript_aws_lambda_os_command_injection /reference/rules/javascript_lang_os_command_injection
 /reference/rules/javascript_aws_lambda_query_injection /reference/rules/javascript_third_parties_dynamodb_query_injection
+
+/reference/rules/ruby_lang_weak_encryption /reference/rules
```

Comment:
@markmichon wondering if it's better (less confusing) to redirect to the rules index page as there's no single replacement here - or do we prefer to select one of the encryption rules?

Reply:
Normally it's not great practice, but for this use case it should be fine.